### DISCUSSION TITLE
Wuuk SD Wifi Config

### DISCUSSION THREAD

**Discussion by nasprouse** [themactep/thingino-firmware#199] [General]
What the is the proper way to configure my Wuuk SSID/Password using the SD card? 

There is conflicting info:

1. https://github.com/themactep/thingino-firmware/wiki/Configuring-Wi%E2%80%90Fi-Access#wireless-credentials-on-an-sd-card
2. https://github.com/themactep/thingino-firmware/wiki/Camera-configuration
3. https://github.com/themactep/thingino-firmware/blob/master/environment/wuuk-y0510.uenv.txt

**Comment by nasprouse** [comment]
Now that this has moved from issues to discussions, any guidance on the proper procedure. The Wiki has conflicting info on how to update the SSID using the SD card. 

The files names to put the SD card are different, the content of the files are different, and the referenced uenv file doesn't have a SSID/password text entry.

**Reply by gtxaspec** [reply]
have you tried `uenv.txt`?
Just asking, why would you use the SD card? Have you tried the WiFi portal?

**Comment by themactep** [comment]
Not a conflicting info. `uenv.txt` and `uEnv.txt` are the same file on a fat partiton which is not case-sensitive.
